tetris.py

```diff
@@ -166,19 +175,20 @@ def block_mover(self):
         Moves the current block downwards one square on the canvas
         """
         if any(
-            (x, y + 1) in self.landed_blocks for (x, y) in self.get_current_block()
-        ) or any(y + 1 == game_height for (x, y) in self.get_current_block()):
+            (x, y + 1) in self.landed_blocks for (x, y) in self.get_current_block()) or any(y + 1 == game_height for (x, y) in self.get_current_block()):
             for coord in self.get_current_block():
                 self.landed_blocks.append(coord)
             print(self.landed_blocks)
             self.full_line_clear()
             self.new_block()
         else:
             x, y = self.current_block_center
-            self.current_block_center = (x, y + 1)
+            self.current_block_center = (x, y)
 
     def block_rotator(self, event):
-        print("TODO: should rotate")
+        self.index += 1
+        if self.index > 4:
+            self.index = 1
```

Comment:
These lines are not necessary because of the `%` usage in  `get_current_block`, and can be deleted safely:
```suggestion
```